tests/c/io_test.cpp

```diff
@@ -125,199 +124,117 @@ void test_keyboard_io(void)
             "kkey2, kdown2 sensekey\n"
             "chnset kkey2, \"key2\"\n"
             "chnset kdown2, \"down2\"\n"
-//            "printk2 kkey2\n"
-//            "printk2 kdown2\n"
             "endin \n";
     csoundCompileOrc(csound, instrument);
     csoundReadScore(csound, "i 1 0 1");
     ret = csoundStart(csound);
-    CU_ASSERT(ret == CSOUND_SUCCESS);
+    ASSERT_TRUE (ret == CSOUND_SUCCESS);
     ret = csoundPerformKsmps(csound);
-    CU_ASSERT(ret == CSOUND_SUCCESS);
+    ASSERT_TRUE (ret == CSOUND_SUCCESS);
     MYFLT val = csoundGetControlChannel(csound, "key", &err);
-//    CU_ASSERT(err == CSOUND_SUCCESS);
-    CU_ASSERT_DOUBLE_EQUAL(val, 100.0, 0.001);
+    ASSERT_DOUBLE_EQ (val, 100.0);
```

Comment:
Csound method returns -1 instead of 100. Seems like the API is wrong here.
```
G:\Programming\GitHub\csound\tests\c\io_test.cpp(135): error: Expected equality of these values:
  val
    Which is: -1
  100.0
    Which is: 100
```